csrc/tvm_ffi_utils.h

```diff
@@ -42,9 +42,10 @@ constexpr DLDataType dl_float32 = DLDataType{kDLFloat, 32, 1};
 constexpr DLDataType dl_float64 = DLDataType{kDLFloat, 64, 1};
 constexpr DLDataType dl_float8_e4m3fn = DLDataType{kDLFloat8_e4m3fn, 8, 1};
 constexpr DLDataType dl_float8_e5m2 = DLDataType{kDLFloat8_e5m2, 8, 1};
+constexpr DLDataType dl_float4_e2m1fn = DLDataType{kDLFloat4_e2m1fn, 4, 1};
```

Comment:
torch only has `torch4_e2m1fn_x2`, can we just keep the x2 version? `dl_float4_e2m1fn` will be invalid when we calculate its element_size (in bytes).